src/LCDC.md

```diff
@@ -110,6 +110,8 @@ When Bit 0 is cleared, the background and window lose their priority -
 the sprites will be always displayed on top of background and window,
 independently of the priority flags in OAM and BG Map attributes.
 
+When Bit 0 is set [see background and window to obj priority](Tile_Maps.md#bg-to-obj-priority-in-cgb-mode).
```

Comment:
Considering the behaviour is simply that OBJs always get priority (which is what's described just above), is it necessary to link to there?

Reply:
When bit 0 is clear the OBJs always get priority, but when bit 0 is set the background and window don't always get priority and that's why I link there

Reply:
Alright, that makes sense. How about this?

```suggestion
When Bit 0 is set, pixel priority is resolved [as described here](Tile_Maps.md#bg-to-obj-priority-in-cgb-mode).
```